core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/coprocessor/transactional/TrxRegionEndpoint.java.tmpl

```diff
@@ -359,7 +359,7 @@ CoprocessorService, Coprocessor {
   private static final int DEFAULT_SLEEP = 60 * 1000;
   private static final int DEFAULT_MEMORY_THRESHOLD = 100; // 100% memory used
```

Comment:
I see. Perhaps add a new "private static final int" for your new usage? For example, "private static final int DEFAULT_STARTUP_MEMORY_THRESHOLD = 90;". Just a stylistic nit.